src/main/java/com/dev/cinema/model/dto/order/OrderResponseDto.java

```diff
@@ -0,0 +1,45 @@
+package com.dev.cinema.model.dto.order;
+
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.User;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
Use Dto rather than Ticket itself

Reply:
@fyarkiy @Borisko130 it is even better to use plain `id`s
```suggestion
    private List<Long> ticketIds;
```